modules/server/package.json

```diff
@@ -63,9 +62,7 @@
     "@types/fresh": "^0.5.0",
     "@types/jquery": "^3.5.11",
     "@types/mime-types": "^2.1.1",
-    "@types/mkdirp": "^1.0.1",
-    "@types/webpack": "^5.28.0",
-    "@types/webpack-dev-server": "^4.5.0"
```

Comment:
I removed these as both webpack and webpack-dev-server now include types in their packages.